docs/download.md

```diff
@@ -100,6 +100,10 @@ The list of all supported operations is provided [here](supported_ops.md).
 For a detailed list of changes, please refer to the
 [CHANGELOG](https://github.com/NVIDIA/spark-rapids/blob/main/CHANGELOG.md). 
 
+**_Note:_** Using the CUDA 450.80.02 driver in combination with the CUDA 10.1 or 10.2 toolkit may
```

Comment:
```suggestion
**_Note:_** Using the NVIDIA driver 450.80.02 in combination with the CUDA 10.1 or 10.2 toolkit may
```

Reply:
If we use "NVIDIA driver" term which can match the download link's term. This can avoid confusing between "CUDA toolkit" version vs "CUDA driver" version.
And also this is what mentioned in https://docs.nvidia.com/deploy/cuda-compatibility/index.html#cuda-intro

Reply:
Updated to refer to the Nvidia driver.